ios/FluentUI/Vnext/Button/MSFButton.swift

```diff
@@ -58,29 +58,12 @@ import UIKit
 
             strongSelf.action?(strongSelf)
         }
-        setupLargeContentViewer()
     }
 
     required public init?(coder: NSCoder) {
         preconditionFailure("init(coder:) has not been implemented")
     }
 
-    private func setupLargeContentViewer() {
```

Comment:
We should keep this while we support iOS 14.

Reply:
@harrieshin Thoughts?

Reply:
I think when we were talking about it, we were saying we should just switch to the new API, but I could be misremembering.

Reply:
@huwilkes That's correct. Consensus was that we will support full a11y on the latest version of iOS, which is currently iOS 15.